tests/ballerina-unit-test/src/test/resources/test-src/h2/h2_actions_test.bal

```diff
@@ -188,81 +189,102 @@ function testUpdateInMemory() returns (int, string) {
+            username: "SA",
+            password: "",
+            poolOptions: { maximumPoolSize: 1 }
+        });
     return selectFunction(testDB);
 }
 
 function testInitWithDbOptions() returns (int[]) {
-    endpoint h2:Client testDB {
+    h2:Client testDB = new({
         path: "./target/H2Client/",
         name: "TestDBH2",
         username: "SA",
         password: "",
         poolOptions: { maximumPoolSize: 1 },
         dbOptions: { "IFEXISTS": true, "DB_CLOSE_ON_EXIT": false, "AUTO_RECONNECT": true, "ACCESS_MODE_DATA": "rw",
             "PAGE_SIZE": 512 }
-    };
+    });
     return selectFunction(testDB);
 }
 
 function testInitWithInvalidDbOptions() returns (int[]) {
-    endpoint h2:Client testDB {
-        path: "./target/H2Client/",
-        name: "TestDBH2",
-        username: "SA",
-        password: "",
-        poolOptions: { maximumPoolSize: 1 },
-        dbOptions: { "IFEXISTS": true, "DB_CLOSE_ON_EXIT": false, "AUTO_RECONNECT": true, "ACCESS_MODE_DATA": "rw",
-            "PAGE_SIZE": 512, "INVALID_PARAM": -1 }
-    };
+    h2:Client testDB = new({
+            path: "./target/H2Client/",
+            name: "TestDBH2",
+            username: "SA",
+            password: "",
+            poolOptions: { maximumPoolSize: 1 },
+            dbOptions: { "IFEXISTS": true, "DB_CLOSE_ON_EXIT": false, "AUTO_RECONNECT": true, "ACCESS_MODE_DATA": "rw",
+                "PAGE_SIZE": 512, "INVALID_PARAM": -1 }
+        });
     return selectFunction(testDB);
 }
 
-function testReInitEndpoint() returns int {
-    endpoint h2:Client testDB {
-        path: "./target/H2Client/",
-        name: "TestDBH2",
-        username: "SA",
-        password: "",
-        poolOptions: { maximumPoolSize: 1 }
-    };
-
-    h2:ClientEndpointConfiguration config = {
-        path: "./target/H2Client/",
-        name: "TestDBH2",
-        username: "SA",
-        password: "",
-        poolOptions: { maximumPoolSize: 1 }
-    };
-
-    testDB.stop();
-
-    testDB.init(config);
-
-    var dt = testDB->select("select 1", Result);
+//function testReInitEndpoint() returns int {
+//    h2:Client testDB = new({
+//            path: "./target/H2Client/",
+//            name: "TestDBH2",
+//            username: "SA",
+//            password: "",
+//            poolOptions: { maximumPoolSize: 1 }
+//        });
+//
+//    h2:ClientEndpointConfiguration config = {
+//        path: "./target/H2Client/",
+//        name: "TestDBH2",
+//        username: "SA",
+//        password: "",
+//        poolOptions: { maximumPoolSize: 1 }
+//    };
+//
+//    testDB.stop();
+//
+//    testDB.init(config);
+//
+//    var dt = testDB->select("select 1", Result);
+//    int count = -1;
+//    if (dt is table) {
+//        while (dt.hasNext()) {
+//            var rs = <Result>dt.getNext();
+//            if (rs is Result) {
+//                count = rs.val;
+//            }
+//        }
+//    }
+//    testDB.stop();
+//
+//    return count;
+//}
```

Comment:
Why this  code commented?

Reply:
This test needs to be removed since re-init is no longer possible. Will remove it and update.